tests/network/expose.go

```diff
@@ -396,26 +466,35 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 					Expect(node.Status.Addresses).ToNot(BeEmpty())
 					nodeIP := node.Status.Addresses[0].Address
 
-					if ipFamily == k8sv1.IPv6Protocol {
-						ipv6NodeIP, err := resolveNodeIPAddrByFamily(
+					var ipv6NodeIP string
+					if ipFamily == ipv6 || ipFamily == dualIPv4Primary || ipFamily == dualIPv6Primary {
```

Comment:
You still use two different conditions which have identical results:
- `ipFamily == ipv6 || ipFamily == dualIPv4Primary || ipFamily == dualIPv6Primary`
- `ipFamily != ipv4`

And vice versa.
Maybe if you extract it to a helper function it will help avoid this.

Reply:
done